packages/hmr-plugin/src/internal/hmr-options-builder.ts

```diff
@@ -1,29 +1,26 @@
-import { isJIT } from '@ngxs/store/internals';
-
 import { NgxsHmrOptions } from '../symbols';
-import { ivyEnabledInDevMode } from '../../../store/src/ivy/ivy-enabled-in-dev-mode';
+import { isDevMode } from '@angular/core';
 
 export class HmrOptionBuilder {
   public readonly deferTime: number;
   public readonly autoClearLogs: boolean;
+  public readonly isIvyMode: boolean;
 
-  constructor({ deferTime, autoClearLogs }: NgxsHmrOptions) {
+  constructor({ deferTime, autoClearLogs, isIvyMode }: NgxsHmrOptions) {
     this.deferTime = deferTime || 100;
     this.autoClearLogs = autoClearLogs === undefined ? true : autoClearLogs;
-    this.validateIvyMode();
+    this.isIvyMode = isIvyMode === undefined ? HmrOptionBuilder.isIvy() : isIvyMode;
+  }
+
+  private static isIvy(): boolean {
```

Comment:
What is the method for since you use it once in 1 place? And why it's `static`?

Reply:
updated